### PR TITLE
[1LP][RFR]V2v migration with special chars name

### PR DESCRIPTION
**Pull request by kedark3**
Purpose or Intent
=================

__Adding tests__v2v migration where migration plan name includes only special characters.

{{pytest: cfme/tests/v2v/test_v2v_migrations.py --use-provider rhv42 --use-provider vsphere65-nested --provider-limit 2 -vvvv --long-running -k 'test_migration_special_char_name' }}

### CODE REVIEW

**Comment by kedark3** [comment]
To be merged after #7866

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e28ef5a3822b657d9134aca1575d154659e2402b:

`cfme/tests/v2v/test_v2v_migrations.py`:
- :red_circle: Line 64:5: **F841** *local variable 'src_vm_ip_addr' is assigned to but never used*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/v2v/test_v2v_migrations.py [hunk 1]
```diff
@@ -506,3 +502,50 @@ def _is_migration_started():
     for vm in vms:
         soft_assert(request_details_list.is_successful(vm) and
          not request_details_list.is_errored(vm))
+
+
+@pytest.mark.parametrize('form_data_vm_obj_single_datastore', [['nfs', 'nfs', rhel7_minimal]],
+                        indirect=True)
+def test_migration_special_char_name(request, appliance, v2v_providers, host_creds, conversion_tags,
+                                    form_data_vm_obj_single_datastore):
```
Comment (sshveta) [inline cfme/tests/v2v/test_v2v_migrations.py]: Question : test name says " test_migration_special_char_name" . 
Can you please add a doc string explaining what and how is the test testing special chars name ?

Comment (kedark3) [reply cfme/tests/v2v/test_v2v_migrations.py]: Added.

File: cfme/tests/v2v/test_v2v_migrations.py [hunk 2]
```diff
@@ -506,3 +502,53 @@ def _is_migration_started():
...
+    src_vm_obj = form_data_vm_obj_single_datastore.vm_obj[0]
+    wait_for(lambda: src_vm_obj.ip_address is not None,
+        message="Waiting for VM to display IP in CFME",
+        fail_func=src_vm_obj.refresh_relationships,
+        delay=5, timeout=300)
+
+    migration_plan_collection = appliance.collections.v2v_plans
+    # fauxfactory.gen_special() used here to create special character string e.g. #$@#@
+    migration_plan = migration_plan_collection.create(
+        name="{}".format(fauxfactory.gen_special()), description="desc_{}"
```
Comment (mshriver) [inline cfme/tests/v2v/test_v2v_migrations.py]: fauxfactory returns a string, but a safe wrapper here doesn't hurt.

**Pull request merged**